tests/sample_data/transform.py

```diff
@@ -0,0 +1,369 @@
+import datetime
+import json
+
+import pytz
+import requests
+
+
+def parse_build_platform(name, os_field="os_name"):
+    platform = ""
+    os_name = ""
```

Comment:
`os_name` does not get changed and can be set as literal empty string in the `return` statement. Similar for `arch`.